src/ParticleDistributions/SuperParticleDistributions.jl

```diff
@@ -0,0 +1,145 @@
+"""
+  particle mass distribution module
+
+Particle mass distribution functions for microphysical process modeling:
+  - computation of densities of distributions
+  - sampling from densities
+"""
```

Comment:
Is this file your original code that you intended to merge with the other ParticleDistribution file? Do you want to keep it?

Reply:
Thanks for pointing that out, I deleted it.